docs/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Add Appropriate Title
+**Models:**
+- Add the changes you did to the model.
+- Add the validations to did to the model.
+- Add anything else you did to the model.
+
+**Controllers:**
+- Add the changes you did to the controller.
+- Add anything else you did to the model.
+
+### Rspec
+**FactoryBot:**
+- Add the new factories you created.
+- Add anything else you did to the factories.
+
+**ModelTesting:**
```

Comment:
maybe should add checklist with options test model/test  request/test feature/  tests not required?

Reply:
I was thinking about it but what if you don't have to do any testing you can just delete the lines I wrote but I can add it

Reply:
done